docroot-goodies/skeleton/templates/page.tpl.php

```diff
@@ -0,0 +1,127 @@
+        <img src="<?php print $logo; ?>" alt="<?php print t('Home'); ?>" />
+      </a>
+    <?php endif; ?>
+
+    <?php if ($site_name || $site_slogan): ?>
+      <div class="name-and-slogan">
+        <?php if ($site_name): ?>
+          <?php if ($title): ?>
+            <div class="site-name"><strong>
+              <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+            </strong></div>
+          <?php else: /* Use h1 when the content title is empty */ ?>
+            <h1 class="site-name">
+              <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+            </h1>
+          <?php endif; ?>
+        <?php endif; ?>
+
+        <?php if ($site_slogan): ?>
+          <div class="site-slogan"><?php print $site_slogan; ?></div>
+        <?php endif; ?>
+      </div> <!-- /.name-and-slogan -->
+    <?php endif; ?>
+
+    <?php print render($page['header']); ?>
+
+  </div>
+</section> <!-- /.main-header -->
+
+<?php if ($main_menu || $secondary_menu): ?>
+  <nav class="main-navigation">
+    <div class="main-navigation-inner">
+      <?php print theme('links__system_main_menu', array('links' => $main_menu, 'attributes' => array('class' => array('main-menu')), 'heading' => t('Main menu'))); ?>
+      <?php print theme('links__system_secondary_menu', array('links' => $secondary_menu, 'attributes' => array('class' => array('secondary-menu')), 'heading' => t('Secondary menu'))); ?>
+    </div> <!-- /.main-navigation-inner -->
+  </nav> <!-- /.main-navigation -->
+<?php endif; ?>
+
+<?php if ($breadcrumb): ?>
+  <div class="breadcrumb"><?php print $breadcrumb; ?></div>
+<?php endif; ?>
+
+<?php if ($messages): ?>
+  <div class="system-messages">
+    <?php print $messages; ?>
+  </div>
+<?php endif; ?>
+
+<div class="main-content-wrapper clearfix">
+
+  <main id="main-content" class="main-content">
+    <div class="main-content-inner">
+      <?php if ($page['highlighted']): ?>
+        <div class="highlighted">
+          <?php print render($page['highlighted']); ?>
+        </div>
+      <?php endif; ?>
+
+      <?php print render($title_prefix); ?>
+
+      <?php if ($title): ?>
+        <h1 class="title page-title"><?php print $title; ?></h1>
+      <?php endif; ?>
+
+      <?php print render($title_suffix); ?>
+
+      <?php if ($tabs): ?>
+        <div class="tabs">
+          <?php print render($tabs); ?>
+        </div>
+      <?php endif; ?>
+
+      <?php print render($page['help']); ?>
+
+      <?php if ($action_links): ?>
+        <ul class="action-links">
+          <?php print render($action_links); ?>
+        </ul>
+      <?php endif; ?>
+
+      <?php print render($page['content']); ?>
+
+      <?php print $feed_icons; ?>
+    </div>
+  </main><!-- /.main-content -->
+
+  <?php if ($page['sidebar_first']): ?>
+    <aside class="sidebar sidebar-first">
```

Comment:
I think `<section>` is correct as opposed to aside, since the contents of the sidebar may not directly relate to the contents of main.